### DISCUSSION TITLE
Text Formatting / Markdown not showing.

### DISCUSSION THREAD

**Discussion by fredofredofred** [ransome1/sleek#663] [General]
Hi,

I am running Windows 10 with Sleek installed via the Microsoft Store. Version is 2.0.8.0.

I have noticed that since updating from an old version the text formatting I has added is not showing. I believe this was based on Markdown previously.

![Sleek markdown](https://github.com/ransome1/sleek/assets/149969915/bad38868-4ba8-40bf-a050-13c487d33170)

Has it been decided to remove this functionality or is it in the process of coming back in version 2?

Personally I like the formatting and would like to see it return. Part of the point of todo.txt is to keep everything human readable. However showing the formatting makes things easier to read and faster to understand.

Let me know if there's anything I can help with. I can't code but I can test stuff out.

Thanks again, I really love this App!

Fred

P.s. As you can probably tell I've just switched from an old version I was running, hence all the feedback.

**Comment by fredofredofred** [comment]
I'll also note that links have stopped appearing with the open button, as shown in the example below taken from version 1.3.1. 

![sleek link](https://github.com/ransome1/sleek/assets/149969915/f7ccd851-9786-441d-8c2c-446d06256469)

I've realised that this is probably part of the rewriting process of version 2 and you probably know about it already - I'll stop reporting every little change unless you tell me it's helpful!

For now I will temporarily swap back to the old version :-)

Fred

**Reply by ransome1** [reply]
Markdown and link support has only been added a couple of days ago with the release of [2.0.9](https://github.com/ransome1/sleek/releases/tag/v2.0.9). 

Which unfortunately is not in the Windows Store, because I ran into technical troubles uploading the build to the store. Maybe the build was corrupted, but the interface threw some undefined error. Unfortunately I cannot rebuild the build, because I don't have access to a Windows computer and since the GitHub pipeline already ran, I cannot re-run it for the 2.0.9 build :/ 

I'm afraid in Windows Store we need to skip this one and 2.0.10 should be available again.

**Reply by fredofredofred** [reply]
Oh cool, glad to hear it's coming back :-)

I'll wait until the next version.

Thanks,

Fred
